plugins/com.google.cloud.tools.eclipse.sdk/src/com/google/cloud/tools/eclipse/sdk/internal/CloudSdkPreferenceResolver.java

```diff
@@ -39,8 +47,22 @@ public CloudSdkPreferenceResolver() {
 
   @Override
   public Path getCloudSdkPath() {
+    // We only consult the Managed Cloud SDK when it has been explicitly configured, which
+    // is done in CloudSdkPreferences.
+    if (preferences.contains(CloudSdkPreferences.CLOUD_SDK_MANAGEMENT)) {
+      String managementOption = preferences.getString(CloudSdkPreferences.CLOUD_SDK_MANAGEMENT);
+      if (CloudSdkManagementOption.AUTOMATIC.name().equals(managementOption)) {
+        // TODO: Should check and wait on the installation job, if in progress
+        try {
+          return ManagedCloudSdk.newManagedSdk().getSdkHome();
+        } catch(UnsupportedOsException ex) {
```

Comment:
style: `catch (`

Reply:
I wonder why these aren't being picked up by the formatter…